src/olympia/versions/models.py

```diff
@@ -130,6 +134,10 @@ def from_upload(cls, upload, addon, platforms, channel, send_signal=True,
                     source=None, is_beta=False):
         from olympia.addons.models import AddonFeatureCompatibility
 
+        if addon.status == amo.STATUS_DISABLED:
+            raise VersionCreateError(
+                'Addon is Mozilla Disabled; no new versions are allowed.')
+
```

Comment:
Is this supposed to be toggled to the user, logged in some way or do we want this to land in sentry (since I don't see it getting catched anywhere)?